lib/client/ipam.go

```diff
@@ -486,7 +486,7 @@ func (c ipams) assignFromExistingBlock(
 
 		// Update the block using CAS by passing back the original
 		// KVPair.
-		obj.Value = &b.AllocationBlock
+		obj.Value = b.AllocationBlock
```

Comment:
It is already a pointer since `b.AllocationBlock` is a pointer.
`fmt.Printf("%T", obj.Value)` prints `*model.AllocationBlock`